events/EventQueue.h

```diff
@@ -127,8 +127,10 @@ class EventQueue : private mbed::NonCopyable<EventQueue> {
      *  returns, as the event may have already begun executing.
      *
      *  @param id       Unique id of the event
+     *  @return         true  if event was successfully cancelled
```

Comment:
It may be worth adjusting the final paragraph above - "If called while the event queue's dispatch loop is active *in another thread*, ...".  A call made from the same thread as the dispatch loop can always succeed.

Reply:
updated comment